__tests__/components/Comments/Comment.test.js

```diff
@@ -96,7 +97,8 @@ describe('Comment test should run correctly', () => {
         findCommentAndUpdate={findComment}
         updateVote={updateVote}
         time={time}
-      />
+      />,
+      {wrapper: Store}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Store` context provider is now being used as a wrapper in the test render function. This change is likely due to new dependencies on context values within the `Comment` component. Make sure that the `Store` context provider is correctly initialized with all necessary values for these tests.

```diff
-       />
+       />,
+       {wrapper: Store}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->